internal/status/status.go

```diff
@@ -0,0 +1,118 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package status
+
+import (
+	"fmt"
+	"github.com/fatih/color"
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/pkg/flowcli/project"
+	"github.com/onflow/flow-cli/pkg/flowcli/services"
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+type FlagsStatus struct {
+}
+
+var statusFlags = FlagsStatus{}
+
+const (
+	NetworkOnline  = "ONLINE"
+	NetworkOffline = "OFFLINE"
+)
+
+var InitCommand = &command.Command{
+	Cmd: &cobra.Command{
+		Use:   "status",
+		Short: "Display status of network",
+	},
+	Flags: &statusFlags,
+	Run: func(
+		cmd *cobra.Command,
+		args []string,
+		globalFlags command.GlobalFlags,
+		services *services.Services,
+	) (command.Result, error) {
+		// Get network name from global flag
+		network := globalFlags.Network
+		if network == "" {
+			network = "emulator"
+		}
+
+		config := globalFlags.ConfigPath
+		proj, err := project.Load(config)
+
+		if err != nil {
+			return nil, fmt.Errorf("project can't be loaded from specified config path")
+		}
+
+		net := proj.NetworkByName(network)
+		accessNode := net.Host
+
+		err = services.Status.Ping()
```

Comment:
I would change this so the value is not err but something more descriptive like `connectionError` and then just pass that to result. The point of result structs is to figure out how to show the result. So:
```go
connectionError = services.Status.Ping()
return &Result{
	network,
	connectionError,
	accessNode,
}, nil
```

This way you don't need to define two results and in results you can just do:

```go
type Result struct {
	network    string
	connectionError     error
	accessNode string
}


if r.connectionError == nil { // online }
```